apps/dashboard/src/lib/components/Org/SideBar.svelte

```diff
@@ -86,35 +92,39 @@
 
       <ul class="my-5">
         {#each menuItems as menuItem}
-          <a
-            href="{$currentOrgPath}{menuItem.path}"
-            class="text-black no-underline"
-            on:click={toggleSidebar}
-          >
-            <li
-              class="flex items-center py-3 px-4 mb-2 {NavClasses.item} {isActive(
-                $page.url.pathname,
-                `${$currentOrgPath}${menuItem.path}`
-              )
-                ? NavClasses.active
-                : 'dark:text-white'}"
+          {#if menuItem.path !== '/setup' || $isOrgAdmin}
```

Comment:
What's the purpose of this condition?

Reply:
This is to make the setup page visible to only admin